amundsen_application/api/search/v0.py

```diff
@@ -21,151 +20,56 @@
 
 search_blueprint = Blueprint('search', __name__, url_prefix='/api/search/v0')
 
-SEARCH_ENDPOINT = '/search'
+SEARCH_ENDPOINT = '/search_table'
```

Comment:
It appears we don't have a generic search endpoint, but a different endpoint for each resource type. Better named as `SEARCH_TABLE_ENDPOINT`.

Reply:
Will do.